lib/membrane_ffmpeg_video_filter/text_overlay.ex

```diff
@@ -8,13 +8,25 @@ defmodule Membrane.FFmpeg.VideoFilter.TextOverlay do
   Additionally, the element has to receive proper caps with picture format and dimensions.
   """
   use Membrane.Filter
+
+  require Membrane.Logger
+
   alias __MODULE__.Native
-  alias Membrane.Buffer
   alias Membrane.Caps.Video.Raw
 
   def_options text: [
                 type: :binary,
-                description: "Text to be displayed on video"
+                description:
+                  "Text to be displayed on video. Either text or text_stream must be provided",
+                default: nil
+              ],
+              text_stream: [
+                type: :list,
+                spec: [{{Time.t(), Time.t() | :infinity}, binary()}],
```

Comment:
For printable strings, you should use `String.t()` as a typespec instead of binary (They are equivalent, but it documents the intent better)